roles/confluent.kafka_broker/tasks/dynamic_groups.yml

```
@@ -0,0 +1,49 @@
+---
+- name: Load server.properties
+  slurp:
+    src: "{{ kafka_broker.config_file }}"
+  register: slurped_properties
+
+# Zookeeper shell cannot read server.properties if secrets protection is enabled. Creates a non encrypted file
+- name: Create Zookeeper TLS Client Config
+  template:
+    src: templates/zookeeper-tls-client.properties.j2
```

do you need the templates/ here?

```suggestion
    src: zookeeper-tls-client.properties.j2
```

---

Maybe it is not needed, but I think I had issues with it before - using the template module. According to the documentation, I think you are correct: https://docs.ansible.com/ansible/latest/user_guide/playbook_pathing.html#the-magic-of-local-paths